ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage.hs

```diff
@@ -13,18 +12,9 @@ import           Test.Tasty (TestTree, testGroup)
 --
 
 tests :: FilePath -> TestTree
-tests tmpDir = testGroup "Storage" $
-    -- The FS tests fail for darwin on CI, see #352. So disable them for now.
```

Comment:
Is this still applicable? In `fs-sim`, we test for all OS's

Reply:
I don't think so

Reply:
Issue https://github.com/input-output-hk/ouroboros-network/issues/532 is still applicable. See this Cicero CI run: https://cicero.ci.iog.io/run/193e7c09-e8d6-48b1-84d9-4a2d69daa6b9.

I'll make sure the tests are not run on Darwin. We should probably revive and fix this issue before publishing `fs-sim` as a package.